packages/grafana-data/src/valueFormats/dateTimeFormatters.ts

```diff
@@ -188,6 +188,17 @@ export function toDays(size: number, decimals?: DecimalCount): FormattedValue {
   }
 }
 
+const units = [
```

Comment:
Uber nit: 
```suggestion
const UNITS = [
```

and please move it to the top of the file

Reply:
sure :)

though i usually prefer to keep consts that were extracted purely for optimization reasons near the functions where they lived.